portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,78 @@
 
 package require json::write
 
+proc format_strings {strings} {
+    if {[llength $strings] > 1} {
+        foreach element $strings {
+            lappend new_strings "\"$element\""
```

Comment:
I believe it makes sense to use `::json::write string $element` here?

Reply:
Yes, it can be used with lappend.

Reply:
Yes, that was what my previous comment on this line was getting at.

Reply:
> Yes, that was what my previous comment on this line was getting at.

Sorry, I had misinterpreted it that time.